compose.go

```diff
@@ -76,29 +75,25 @@ func (f LocalDockerComposeOptionsFunc) ApplyToLocalCompose(opts *LocalDockerComp
 	f(opts)
 }
 
-// NewLocalDockerCompose returns an instance of the local Docker Compose, using an
-// array of Docker Compose file paths and an identifier for the Compose execution.
-//
-// It will iterate through the array adding '-f compose-file-path' flags to the local
-// Docker Compose execution. The identifier represents the name of the execution,
-// which will define the name of the underlying Docker network and the name of the
-// running Compose services.
-func NewLocalDockerCompose(filePaths []string, identifier string, opts ...LocalDockerComposeOption) *LocalDockerCompose {
+func NewDockerCompose(
```

Comment:
This signature change comes with a breaking change :) Thankfully we are not in 1.x version, so we can change contracts

Reply:
Oh oh wait. Sorry, but I thought this line was changing the `NewLocalDockerCompose` method, but it's not: it's adding a new generic method.